src/views/CoCurricularTranscript/index.js

```diff
@@ -242,7 +245,8 @@ export default class Transcript extends Component {
   };
 
   render() {
-    if (this.props.authentication) {
+    console.log(this.state.profile.PersonType);
```

Comment:
Please remove this console log before merging.

Reply:
thaanks!